app/javascript/profilePreviewCards/UserMetadata.jsx

```diff
@@ -0,0 +1,76 @@
+import { h, Fragment } from 'preact';
+import { memo } from 'preact/compat';
+
+/**
+ * Component which renders the user metadata detail in a profile preview card.
+ *
+ * @param {string} email The user's email (if set to be publicly displayed)
+ * @param {string} location The user's location
+ * @param {string} created_at The user's join date string
+ * @param {string} education The user's education detail
+ * @param {string} work The user's work details
+ */
```

Comment:
Documentation (blocking): These are properties of the props object. Intellisense/JSDoc will not work. Prefix all parameters with `props.`

```suggestion
/**
 * Component which renders the user metadata detail in a profile preview card.
 *
 * @param {object} props
 * @param {string} props.email The user's email (if set to be publicly displayed)
 * @param {string} props.location The user's location
 * @param {string} props.created_at The user's join date string
 * @param {string} props.education The user's education detail
 * @param {string} props.work The user's work details
 */
```

Before

![Incorrect JSDoc comments for the UserMetaData component](https://user-images.githubusercontent.com/833231/129055284-ef8f5853-f793-4918-888a-4f8a16aeeebb.png)

After

![Correct JSDoc comments for the UserMetaData component](https://user-images.githubusercontent.com/833231/129055163-44b82619-e8ba-457a-9625-607c91c35ae7.png)